### PR TITLE
S prefix remove noarch

### PR DESCRIPTION
**Pull request by sgn**

### CODE REVIEW

File: srcpkgs/stk/template [hunk 1]
```diff
@@ -66,7 +66,6 @@ post_install() {
 
 stk-data_package() {
```
Comment (ericonr) [inline srcpkgs/stk/template:67]: Why not kill this data package?

**Review by ericonr — APPROVED** [review]

File: srcpkgs/stk/template (hunk 1 above)
Comment (sgn) [reply srcpkgs/stk/template:67]: I don't know. Maybe we should kill this -data too.

**Comment by sgn** [comment]
932d410a9d5d9f3034908412ebff70f636bf64d7

File: srcpkgs/stk/template (hunk 1 above)
Comment (sgn) [reply srcpkgs/stk/template:67]: No, there're other packages depend on `stk-data`. I'm better to not push on a rush.

**Pull request closed without merging**